tuned/ppd/ppd.conf

```diff
@@ -1,9 +1,13 @@
 [main]
 # The default PPD profile
 default=balanced
+battery_detection=true
 
 [profiles]
 # PPD = TuneD
 power-saver=powersave
+power-saver-battery=powersave
 balanced=balanced
+balanced-battery=balanced-battery
 performance=throughput-performance
+performance-battery=throughput-performance
```

Comment:
I'm not a fan of the "-battery" suffix syntax, as it introduces redundancy and requires many unappealing changes. As of now, it also breaks the `propertyChanged` signal, sending the profile name with the "battery" suffix included. Could we instead make it two sections?
```ini
[profiles]
power-saver=powersave
balanced=balanced
performance=throughput-performance

[battery]
balanced=balanced-battery
```
When the system is unplugged from AC, we could check whether the current profile is in the `battery` section and if yes, apply the TuneD profile from there.

Reply:
Makes perfect sense.  I'll modify.